### ISSUE TITLE
cannot pair the camera to my wifi

### ISSUE THREAD

**Issue by ICrashWindows12** [DavidVentura/cam-reverse#44]
Model: A9
Manufacturer: Telurahl (it states on packaging ) (I also don't know if I got the manufacturer name correct)
A9 AP starts with: Nax (short for naxclow)
My laptop OS: ChromeOS with Linux Container Bullseye/Crostini
CPU arch: ARM64
I cannot pair for some reason. The IP address starts at 192.168.169.1 but when I pair it states it looks for devices at 192.168.1.255. I am connected to the AP of the camera. is there any way to change. Also is this compatible with chromeOS running Linux debian vm (the command line) 

2024-09-21T11:06:16.808Z [error] Node version 12 is not supported, may malfunction
2024-09-21T11:06:16.811Z [info] Will configure any devices found to join Optus_XXXXXX
2024-09-21T11:06:16.848Z [info] Searching for devices on 192.168.1.255

**Comment by ICrashWindows12** [comment]
does changing the bin.cjs file even do anything. I edited it with `sudo nano bin.cjs` then I found the discovery ips section and changed from 192.168.1.255 to 192.168.169.255, then I tried 192.168.169.1 and still nothing

**Comment by ICrashWindows12** [comment]
I executed the command like this `node bin.cjs pair --ssid Optus_XXXXX --password <Password>` obviously with the password after I connected to the devices AP.

**Comment by ICrashWindows12** [comment]
I'm going to see if I can connect the X5 mini camera

**Comment by ICrashWindows12** [comment]
also NAX

**Comment by ICrashWindows12** [comment]
nope. maybe the restrictions on chromeOS's Linux container does not allow it to discover devices. also the one by int is just so confusing

**Comment by DavidVentura** [comment]
can you make a network capture while connected to the camera's AP?

**Comment by ICrashWindows12** [comment]
how do I make a network capture

**Comment by ICrashWindows12** [comment]
> can you make a network capture while connected to the camera's AP?

while connected to the cameras AP?

**Comment by ICrashWindows12** [comment]
> can you make a network capture while connected to the camera's AP?

How do I make a network capture. I just found out about network capturing and I'm new to it

**Comment by ICrashWindows12** [comment]
> can you make a network capture while connected to the camera's AP?

do I run the script along with wireshark

**Comment by ICrashWindows12** [comment]
> can you make a network capture while connected to the camera's AP?

![Capturing from Wi-Fi 22_09_2024 12_10_12 AM](https://github.com/user-attachments/assets/ce98666f-1f02-482d-a682-7dbab74d0c1b)

**Comment by ICrashWindows12** [comment]
i did it on windows because I couldn't figure it on chromeOS

**Comment by ICrashWindows12** [comment]
> can you make a network capture while connected to the camera's AP?

I did already. was it supposed to be after I ran the script

**Comment by ICrashWindows12** [comment]
> can you make a network capture while connected to the camera's AP?

PLEASE RESPOND. I may have found something
![Screenshot 2024-09-23 21 37 44](https://github.com/user-attachments/assets/38c470a4-91f6-4eea-a411-daee586af02c)
![Screenshot 2024-09-23 21 37 24](https://github.com/user-attachments/assets/7406455c-258f-43dc-b92a-44fea0645997)
. I don't know if it is useful info.

**Comment by rundekugel** [comment]
1st: David: thanks for your work!

similar problem here.
the param `--discovery_ip 192.168.1.1`  is not working for me with command 'pair'
changed `discovery_ips` in bin.cjs
camera is found, but :

```
Logging in to camera BATE637140UAGEB
2024-10-20T18:20:15.045Z [info] Scanning for Wifi networks on BATE637140UAGEB -- this may time out
2024-10-20T18:20:22.690Z [info] Camera BATE637140UAGEB at 192.168.1.1 already discovered, ignoring
2024-10-20T18:20:23.989Z [info] Camera BATE637140UAGEB at 192.168.1.1 already discovered, ignoring
2024-10-20T18:20:25.399Z [warning] Camera BATE637140UAGEB timed out
2024-10-20T18:20:25.399Z [info] Disconnected from camera BATE637140UAGEB at 192.168.1.1
2024-10-20T18:20:25.400Z [info] Camera BATE637140UAGEB disconnected
2024-10-20T18:20:26.991Z [info] Discovered camera BATE637140UAGEB at 192.168.1.1
2024-10-20T18:20:27.045Z [info] Logging in to camera BATE637140UAGEB
```
Then I reconnected wifi, started bin.cjs again... it worked.

**Comment by kohend** [comment]
Changed in bin.cjs and it works for me as well, thanks @rundekugel
